src/models/observation.js

```diff
@@ -76,7 +76,7 @@ module.exports = (sequelize, DataTypes) => {
         this.setDataValue('phenomenonTime', new Date(value).toISOString());
       }
     },
-    result: { type: DataTypes.JSONB, allowNull: false },
+    result: { type: DataTypes.STRING, allowNull: false },
```

Comment:
This means that the type of Observation results can only be a string and that's not what's defined by the spec :(

Should we process the value before and after storing and retrieving it from the db? We can do it with a sequelize getter and setter. The data type of result is defined by the associated Datastream, **iff** we could access it from the setter/getter we could process the value according [this table](http://docs.opengeospatial.org/is/15-078r6/15-078r6.html#table_12)